auth/Cargo.toml

```diff
@@ -10,15 +10,15 @@ categories = ["authentication", "web-programming"]
 edition = "2021"
 
 [dependencies]
+async-trait = "0.1"
+backoff = { version="0.4", features = ["tokio"] }
 base64 = "0.13"
 chrono = { version = "0.4", features = ["serde"] }
-reqwest = {version = "0.11", features = ["json"] }
+http = "0.2"
+reqwest = { version = "0.11", default-features = false, features = ["json", "rustls-tls"] }
```

Comment:
This line is the only actual change. Other lines are just from alpha sorting the dependency list.